core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -145,21 +145,6 @@ trait RichCompilerControl extends CompilerControl with RefactoringControl with C
   def askReloadAndTypeFiles(files: Iterable[SourceFile]) =
     askOption(reloadAndTypeFiles(files))
 
-  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[RangePosition]] = {
```

Comment:
Ah, yes.
Though augmenting the results is going to require some more work.
Since now we first query the analyzer to obtain the fqn and the reroute to indexer so that it can find it's usages, should we also pass along the results we have from the loaded files?